lib/quby/questionnaires/entities/flag.rb

```diff
@@ -18,6 +18,16 @@ def initialize(key:,
         def if_triggered_by(answer_flags)
           yield if answer_flags[key] == trigger_on
         end
+
+        def to_codebook(_options = {})
+          output = []
+          output << "#{key} flag"
+          output << " 'true' - #{description_true}"
+          output << " 'false' - #{description_false}"
+          output << " '' (leeg) - Vlag niet ingesteld, informatie onbekend"
```

Comment:
Je wilt zeggen dat `flag.trigger_on == true && flag.value == false` niet exact dezelfde lijst geeft als `flag.value == nil` ?

Of dat dit voor de gebruiker niet belangrijke informatie is om te weten dat je die instanties hetzelfde kan behandelen?